package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@folio/orders",
-  "version": "5.1.0",
+  "version": "6.0.0",
```

Comment:
why do you think it's a breaking change?

Reply:
We're discontinuing support for `orders <12.1` and `order-lines <3.3`. Interface requirements are a part of the public API. This change is backward incompatible, as `ui-orders` will no longer function in an environment that provides `orders <12.1` or `order-lines <3.3`.

Reply:
> If you implement any backwards-incompatible change to any API, or drop any API at all, increment the module version to 3.0.0

[Guidelines for Contributing Code](https://dev.folio.org/guidelines/contributing/#module-implements-multiple-interfaces)